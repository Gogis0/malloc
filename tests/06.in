free 1044
alloc -2
alloc 21
free 1267
free 235
free 564
alloc 132
alloc -14
free 256
free 903
alloc 21
free 795
free 494
free 626
free 1128
alloc 101
alloc 19
free 1794
alloc -71
alloc 211
end
