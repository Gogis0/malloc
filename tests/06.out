-1
-1
20
-1
-1
-1
61
-1
-1
-1
213
-1
-1
-1
-1
254
375
-1
-1
414
